README.md

```diff
@@ -1971,4 +1971,244 @@ You can create a `./gradlew checkstyle` command using the following:
 task checkstyle {
     dependsOn(checkstyleMain, checkstyleTest)
 }
-```
\ No newline at end of file
+```
+
+## TDD with Fibonacci
+
+As with all Test Driven Development, the key things to remember are:
+
+1. Red: Write as little code as possible to make your test fail.
+1. Green: Write as little code as possible to make your test pass.
+1. Refactor: Optimise your code for readability without changing its behaviour.
+
+First, create a test class that calls the intended target class:
+
+``` java
+public class FibonacciTest {
+    
+    Fibonacci target = new Fibonacci();
+
+}
+```
+
+Then create that target class.
+
+``` java
+public class Fibonacci {
+}
+```
+
+Second, add a simple method to the test class.
+
+``` java
+public class FibonacciTest {
+
+    public void shouldBe() {
+        Fibonacci target = new Fibonacci();
```

Comment:
I don't see why we make this field a local variable at this point. Let's just keep it as a field.